### PR TITLE
[BEAM-4571] RedisIO support for write using SET operation

### PR DESCRIPTION
**Pull request by henryken**
Add support for set, lpush, and rpush Redis operations, including setting expire time

R: @jbonofre

### CODE REVIEW

**Comment by henryken** [comment]
retest this please

**Comment by henryken** [comment]
retest this please

**Comment by henryken** [comment]
Hi, is there any comment on this pull request?

File: sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java [hunk 1]
```diff
@@ -52,32 +58,101 @@ public void after() throws Exception {
   }
 
   @Test
-  public void testWriteRead() throws Exception {
+  public void testWriteRead_append() throws Exception {
```
Comment (jbonofre) [inline sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java]: For consistency, I would use `testWriteReadAppend()`.

File: sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java [hunk 2]
```diff
@@ -52,32 +58,101 @@ public void after() throws Exception {
...
+            RedisIO.read()
+                .withEndpoint(REDIS_HOST, embeddedRedis.getPort())
+                .withKeyPattern("foobar*"));
     PAssert.thatSingleton(readNotMatch.apply(Count.globally())).isEqualTo(0L);
 
     readPipeline.run();
   }
 
+  @Test
+  public void testWriteRead_set() throws Exception {
```
Comment (jbonofre) [inline sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java]: Same here `testWriteReadSet()`.

**Comment by henryken** [comment]
retest this please

**Comment by henryken** [comment]
Thanks for your review @jbonofre. I've made the changes and the build is good. Let me know if there is anything else.

**Pull request merged**